packages/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemModule.java

```diff
@@ -517,6 +520,38 @@ public void onResponse(Call call, Response response) throws IOException {
     }
   }
 
+  @ExpoMethod
+  public void getContentUriAsync(String uri, Promise promise){
```

Comment:
This line is ill-formatted. Please run files you changed through Android Studio formatter (I have a macro defined that autoformats files on <kbd>Cmd</kbd>+<kbd>S</kbd>)

<img width="690" alt="Zrzut ekranu 2019-06-17 o 15 22 35" src="https://user-images.githubusercontent.com/1151041/59607502-be064980-9113-11e9-908b-481247cad58e.png">